cli/cmd/dashboard.go

```diff
@@ -20,39 +20,44 @@ var dashboardCmd = &cobra.Command{
 	Long:  "Open the Conduit dashboard in a web browser.",
 	RunE: func(cmd *cobra.Command, args []string) error {
 		if proxyPort <= 0 {
-			log.Fatalf("port must be positive, was %d", proxyPort)
+			return fmt.Errorf("port must be positive, was %d", proxyPort)
 		}
 
 		kubectl, err := k8s.NewKubectl(shell.NewUnixShell())
 		if err != nil {
-			log.Fatalf("Failed to start kubectl: %v", err)
+			fmt.Fprintf(os.Stderr, "Failed to start kubectl: %s", err)
+			os.Exit(1)
```

Comment:
curious: in this context, how do you think about the difference between:
```golang
return fmt.Errorf(...
```
and:
```golang
fmt.Fprintf(os.Stderr, ...
os.Exit(1)
```

Reply:
This is a good question. If the run function returns an error, then cobra will print the error text and the usage message for the command that's being run. In most cases that's the right behavior (e.g. like on line 23 of this file, where the `--proxy-port` flag is invalid). But in this case, the usage checks have passed and something else has failed, in which case I think it's preferable to just print the failure and exit.

Reply:
Makes sense, thanks for the explanation!